tests/build/chains.go

```diff
@@ -17,7 +17,9 @@ import (
 	"github.com/redhat-appstudio/e2e-tests/pkg/utils/tekton"
 )
 
-var _ = framework.ChainsSuiteDescribe("Tekton Chains E2E tests", Label("ec", "HACBS"), Pending, func() {
+
+var _ = framework.ChainsSuiteDescribe("[test_id:04] Tekton Chains E2E tests", Label("ec", "HACBS"), func() {
```

Comment:
Note that this test was marked with "Pending" label, i.e. was disabled, because it's not working. That's why the CI is currently failing.